lib/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "AWS_ACCOUNT": "427935540279",
+  "AWS_REGION": "us-east-1",
+  "OIDC_URL": "https://cognito-idp.us-east-1.amazonaws.com/us-east-1_gGfd5y9Fd",
```

Comment:
This file should probably be removed.

Reply:
This _should_ be in `.gitignore` - I could have sworn it was but doesn't appear to be in the committed code.